python/py_helpers.py

```diff
@@ -424,6 +436,20 @@ def _find_conditions(tree):
 
         return [Node(test) for test in _find_conditions(self.tree)]
 
+    # Returs a Boolean indicating if node1 comes before node2
+    def is_ordered(self, node1, node2):
```

Comment:
Please, help me find a better name for this.

Reply:
The name is fine as it is serving the intended purpose 👍️